front/lib/api/assistant/configuration.ts

```diff
@@ -37,6 +37,12 @@ import {
   DEFAULT_TABLES_QUERY_ACTION_NAME,
   DEFAULT_WEBSEARCH_ACTION_NAME,
 } from "@app/lib/api/assistant/actions/names";
+import { fetchBrowseActionsConfigurations } from "@app/lib/api/assistant/configuration/browse";
```

Comment:
nit: the double plural here feels a little weird to me, I would rather `fetchBrowseActionConfigurations`

Reply:
Agree, updated.